spec/services/link_check_report/create_service_spec.rb

```diff
@@ -0,0 +1,217 @@
+require "spec_helper"
+
+RSpec.describe LinkCheckReport::CreateService do
+  let(:user) { FactoryGirl.create(:user) }
+
+  let(:link_check_report) { double(:link_check_report, valid?: nil, save!: nil) }
```

Comment:
Could we use a real link check report here?

Reply:
Ditto previos comment.

Reply:
My concern here is that I can see some of the code was using fields like `link_reportable_type` which no longer exists, if this was a real object, the tests would have caught it.